pkg/cmd/cli/restore/create.go

```diff
@@ -144,6 +144,7 @@ func (o *CreateOptions) Run(c *cobra.Command, f client.Factory) error {
 		return err
 	}
 
-	fmt.Printf("Restore %q created successfully.\n", restore.Name)
+	fmt.Printf("Restore request for %q submitted successfully.\n", restore.Name)
```

Comment:
I feel like this should either say "Restore request <restore.Name>", or "Restore request for <o.BackupName>" - WDYT?

Reply:
I think if/until we change to `ark restore create <name> --backup <backup name>`, the output here must include at least the restore's generated name. Since that includes the backup's name, I think the former (just removing "for") is probably better.

Reply:
Ok, can do that. I used “for” as I was viewing it was the request for the restore object to be created. However, the request isn’t really an object in Ark so I suppose it makes sense to drop the “for”.

Reply:
Message updated.